src/main/java/com/alkemy/ong/entity/Members.java

```diff
@@ -34,7 +34,7 @@ public class Members {
     @Nullable
     private String linkedinUrl;
 
-    @Column(nullable = false)
+    @Column
```

Comment:
Este cambio no corresponde
En el ticket OT296-23 para entidad Member se especifica: image: VARCHAR NOT NULL